### DISCUSSION TITLE
[Project Proposal] Move nexus-casc-plugin to the sonatype-nexus-community organization

### DISCUSSION THREAD

**Discussion by larhauga** [sonatype-nexus-community/.github#8] [Project Proposal]
### What is your Project Idea?

Hi,
over the years there have been many forks of https://github.com/sventschui/nexus-casc-plugin which provides a great way of managing nexus repository with Configuration as Code. I am currently trying to keep github.com/larhauga/nexus-casc-plugin a bit more up-to-date.

Would you consider adding a fork into the sonatype-nexus-community organization?
It would make the project more sustainable, and no longer dependent on individual contributors.

FYI @sonatype-nexus-community/community-leaders

**Comment by eddie-knight** [comment]
Hi @larhauga, thanks for this proposal! The community leaders team has begun to investigate— we'll circle back here to discuss more within the next week.

**Reply by madpah** [reply]
Hey @larhauga - we're just waiting on some internal folk to circle back to us about this request. Apologises we don't have a clear answer and path yet - we haven't forgotten about you!

We're also taking this opportunity to publicly document the process of projects joining the Sonatype Community - https://github.com/sonatype-nexus-community/contribute.sonatype.com/pull/40

**Reply by madpah** [reply]
Hey @larhauga - thanks for bearing with us. In principle, we're happy for the project to be adopted by Sonatype Open Source Community.

Next step would be to have a short Zoom to discuss some of the finer details.

If that's acceptable, can you drop an email to community [AT] sonatype.com and let us know some dates/times next week and week after?

Thanks!
